### PR TITLE
feat(default): Follow commitlint-config-seek conventions

### PR DESCRIPTION
**Pull request by michaeltaranto**
Changing the default PR title and commit message to conform with our [`commitlint-config-seek`](https://github.com/seek-oss/commitlint-config-seek) configuration.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,8 +29,9 @@
       "lockFileMaintenance": {
         "enabled": false
       },
-      "rebaseStalePrs": true,
-      "semanticCommits": true,
+      "semanticCommits": false,
+      "prTitle": "{{semanticCommitType}}({{semanticCommitScope}}): {{#if isPin}}Pin{{else}}{{#if isRollback}}Roll back{{else}}Update{{/if}}{{/if}} {{depName}} to {{#unless isRange}}v{{/unless}}{{#if isMajor}}{{newVersionMajor}}.x{{else}}{{newVersion}}{{/if}}",
```
Comment (markdalgleish) [inline package.json:33]: 😅👍

Comment (kosanna) [reply package.json:33]: 🤕 😂

Comment (markdalgleish) [reply package.json:33]: Just double checked this—love that this is scoped to *all* packages, not just SEEK ones. We should probably call this out in the readme as one of this package's selling points.

File: package.json [hunk 2]
```diff
@@ -50,12 +51,12 @@
...
   },
   "devDependencies": {
-    "@commitlint/cli": "^5.2.8",
+    "@commitlint/cli": "^6.0.2",
     "commitizen": "^2.9.6",
     "commitlint-config-seek": "^1.0.0",
     "cz-conventional-changelog": "^2.1.0",
     "husky": "^0.14.3",
-    "renovate": "10.46.0",
+    "renovate": "10.49.3",
```
Comment (jahredhope) [inline package.json:59]: Not strictly related to this pr but should this be added as a peerDependency as-well?

**Review by jahredhope — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (markdalgleish) [reply package.json:59]: This is only here for testing purposes, to ensure our config is actually valid. Consumers don't need Renovate to be installed.

File: package.json (hunk 1 above)
Comment (michaeltaranto) [reply package.json:33]: Added to the readme 👍

**Review by jahredhope — APPROVED** [review]

**Review by markdalgleish — APPROVED** [review]

**Pull request merged**